src/gretel_client/transformers/transformers/date_shift.py

```diff
@@ -29,6 +31,7 @@ class DateShiftConfig(RestoreTransformerConfig):
     upper_range_days: int = None
     secret: str = None
     tweak: FieldRef = None
+    format: str = None
```

Comment:
- let's use a non-reserved keyword, like `date_format`

- the docstrings will need to be updated with:
  - the new param
  - below the params we'll need to explain exactly what happens if the format is excluded and why the output format could change, and what happens if the format can't be applied, etc. i.e. there shouldn't be any surprises as to what happens when using this based on the possible scenarios

Reply:
On it